job/server/src/main/java/alluxio/master/job/JobMaster.java

```diff
@@ -242,6 +244,23 @@ public void cancel(long jobId) throws JobDoesNotExistException {
     return jobCoordinator.getJobInfoWire();
   }
 
+  /**
+   * Gets summary of the job service.
+   *
+   * @return {@link JobServiceSummary}
+   */
+  public alluxio.job.wire.JobServiceSummary getSummary() {
+    Collection<JobCoordinator> coordinators = mTracker.coordinators();
+
+    List<alluxio.job.wire.JobInfo> jobInfos = new ArrayList<>();
+
+    for (JobCoordinator coordinator : coordinators) {
+      jobInfos.add(coordinator.getJobInfoWire());
+    }
+
+    return new JobServiceSummary(jobInfos);
```

Comment:
This isn't quite true because this is not a lifetime metric. I'll need to rethink how I would get real metrics out especially regarding job completions.

Reply:
Oh I see, on the server side we only keep track up to # of retained jobs?